.idea/codeStyles/Project.xml

```diff
@@ -577,9 +577,7 @@
     <codeStyleSettings language="protobuf">
       <option name="RIGHT_MARGIN" value="80" />
       <indentOptions>
-        <option name="INDENT_SIZE" value="2" />
         <option name="CONTINUATION_INDENT_SIZE" value="2" />
-        <option name="TAB_SIZE" value="2" />
       </indentOptions>
```

Comment:
given the commit message im not sure why these are changed

Reply:
Added more commit description, although I am not 100% certain on the origin of these changes (the IDE did the changes)

Reply:
That seems strange, ideally those would be a seperate commit.

Reply:
Updated